### DISCUSSION TITLE
Using pretrained ResNet with MedicalNET weights

### DISCUSSION THREAD

**Discussion by Meddebma** [Project-MONAI/tutorials#516] [General] [answered]
Hi, 
I am trying to make a classification for 3d data using a pretrained Resnet with MedicalNet weights, when I start to train, I get this error:
Error(s) in loading state_dict for ResNet:
	size mismatch for conv1.weight: copying a param with shape torch.Size([64, 1, 7, 7, 7]) from checkpoint, the shape in current model is torch.Size([64, 3, 7, 7, 7]).

Could you help me find out where the mismatch come from?
Here is the Notebook:
https://github.com/Meddebma/pyradiomics/blob/master/Classification_HBI_outcome.ipynb

Thanks!

**Comment by Nic-Ma** [comment]
Hi @yiheng-wang-nv ,

Could you please help take a look at this ticket?

Thanks in advance.

**Comment by yiheng-wang-nv** [comment, marked as answer]
HI @Meddebma , the default number of input channels of monai's resnet is 3 (https://github.com/Project-MONAI/MONAI/blob/dev/monai/networks/nets/resnet.py#L163), but in the pretrained weights it was 1. If you need to use that pretrained weights, you may need to load the weights except the conv1 layer.
You can check [here](https://github.com/Project-MONAI/MONAI/blob/dev/monai/networks/nets/senet.py#L316) for reference

**Comment by Hogandr** [comment]
Hey @Meddebma,
can you share your working code?
Thanks in advance
